developer-workflow/psrt.rst

```diff
@@ -0,0 +1,148 @@
+will triage the report and if the report seems plausible
+(ie, not spam and for the correct project) will reply with
+instructions on how to report the vulnerability on GitHub.
+
+If the reporter doesn't want to use GitHub's Security Advisories feature
+then the PSRT admins can create a draft report on behalf of the reporter.
+
+Coordinating a vulnerability report
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each report will have a member of the PSRT assigned as the "coordinator".
+The coordinator will be responsible for following the below process and
+will be publicly credited on vulnerability records post-publication.
+
+If a coordinator can't complete the process for any reason (time obligation,
+vacation, etc) they must find a replacement coordinator in the PSRT
+and reassign the vulnerability report appropriately.
+
+Coordinators are expected to collaborate with other PSRT members and core developers
+when needed for guidance on affect-ness, severity, advisory text, and fixes.
+
+**The vulnerability coordination process is:**
+
+* Determine whether the report constitutes a vulnerability. If the report isn't a vulnerability,
+  the reporter should be notified appropriately. Close the GHSA report, the report can be reopened if
+  sufficient evidence that the report is a vulnerability is later obtained.
+
+* After a vulnerability report is accepted, a CVE ID must be assigned. If this is not done
+  automatically, then a CVE ID can be obtained by sending an email to ``cna@python.org``.
+  No details about the vulnerability report need to be shared with the PSF CNA for a CVE ID to be reserved.
+
+* Once a CVE ID is assigned, the CVE ID and acceptance can be shared with the reporter.
+  Use this CVE ID for referencing the vulnerability. Ask the reporter
+  if they'd like to be credited publicly for the report and if so, how they'd like to be credited.
+  Add this information to the GitHub Security Advisory.
+
+* If the report is a vulnerability, determine the severity of the vulnerability. Severity is one of:
+  **Low**, **Medium**, **High**, and **Critical**. Coordinators can use their knowledge of the code, how the code is likely used,
+  or another mechanism like CVSS for determining a severity. Add this information to the GitHub Security Advisory.
+
+* Author the vulnerability advisory text. The advisory must include the following information:
+
+  * Title should be a brief description of the vulnerability and affected component (e.g "Buffer over-read in SSLContext.set_npn_protocols()")
+
+  * Short description of the vulnerability, impact, and the conditions where the affected component is vulnerable, if applicable.
+
+  * Affected versions. This could be "all versions", but if the vulnerability exists in a new feature
+    or removed feature then this could be different. Include versions that are end-of-life in this calculation.
+    (e.g. "Python 3.9 and earlier", "Python 3.10 and later", "all versions of Python")
```

Comment:
Should we be more precise with these? Generally it's going to be "all the latest versions as of the day we disclose", which means we can easily figure out the real list.

Reply:
It might also be useful to describe which versions are (or will be) EOL as of the disclosure date.